CONTRIBUTING.md

```diff
@@ -132,6 +132,37 @@ See [docs/contrib/Dev_Environment.md](docs/contrib/Dev_Environment.md)
 
 - If you are a developer working on Kai and you are updating requirements.txt, you will need to do some manual changes beyond just a `pip freeze &> ./requirements.txt`, we have a few directives that address differences in 'darwin' systems that need to be preserved. These need to be added manually after a 'freeze' as the freeze command is not aware of what exists in requirements.txt. Please consult the diff of changes you are making now from prior version and note the extra directions for `python_version` and or `sys_platform`
```

Comment:
This section on the requirements.txt is no longer relevant and should be removed as we now follow the procedure of placing requirements in requirements.in and generating the requirements.txt with `pip-compile`